### PR TITLE
Update functions for preview

### PR DESCRIPTION
**Pull request by 4D42**
Numpy had some update that make caiman preview not working.

I updated the functions that are being used for the preview from the updated version of caiman package.

### CODE REVIEW

File: CaImAn/caiman_run.py [hunk 1]
```diff
@@ -36,6 +36,9 @@
 logging.basicConfig(level=logging.DEBUG)
 freeze_support()
 
+kwargs = {}
+params_doric = {}
+
```
Comment (4D42) [inline CaImAn/caiman_run.py:41]: I added that because of VScode so kwargs and params_doric are recognized as variable

**Comment by doriclenses** [comment]
> Numpy had some update that make caiman preview not working.
> 
> I updated the functions that are being used for the preview from the updated version of caiman package.

Do we have to update numpy? Shouldn't we use the version of numpy that is in the requirements of caiman?

**Comment by 4D42** [comment]
> > Numpy had some update that make caiman preview not working.
> > I updated the functions that are being used for the preview from the updated version of caiman package.
> 
> Do we have to update numpy? Shouldn't we use the version of numpy that is in the requirements of caiman?

I re installed caiman in anaconda. I did that because I thought it could fixe some problem we had with caiman and also I thought it was a good opportunity to have the last version of caiman ( It was a sort of "mistake" to do that) so now it is using newer version of numpy.

Also before doing that I try to downgrade numpy and I couldn't do that because of dependency and so on....  so I don't really want to start to change numpy version and have thing broken in caiman just because of that.

**Comment by doriclenses** [comment]
> > > Numpy had some update that make caiman preview not working.
> > > I updated the functions that are being used for the preview from the updated version of caiman package.
> > 
> > 
> > Do we have to update numpy? Shouldn't we use the version of numpy that is in the requirements of caiman?
> 
> I re installed caiman in anaconda. I did that because I thought it could fixe some problem we had with caiman and also I thought it was a good opportunity to have the last version of caiman ( It was a sort of "mistake" to do that) so now it is using newer version of numpy.
> 
> Also before doing the pullrequest I did try to downgrade numpy and I couldn't do that because of dependency and so on.... so I don't really want to start to change numpy version and have thing broken in caiman just because of that.

Ok, the caiman library changed the requirements for numpy? Or just numpy released a new version?

**Comment by doriclenses** [comment]
> > > > Numpy had some update that make caiman preview not working.
> > > > I updated the functions that are being used for the preview from the updated version of caiman package.
> > > 
> > > 
> > > Do we have to update numpy? Shouldn't we use the version of numpy that is in the requirements of caiman?
> > 
> > 
> > I re installed caiman in anaconda. I did that because I thought it could fixe some problem we had with caiman and also I thought it was a good opportunity to have the last version of caiman ( It was a sort of "mistake" to do that) so now it is using newer version of numpy.
> > Also before doing the pullrequest I did try to downgrade numpy and I couldn't do that because of dependency and so on.... so I don't really want to start to change numpy version and have thing broken in caiman just because of that.
> 
> Ok, the caiman library changed the requirements for numpy? Or just numpy released a new version?

I see, they don't have any requirements for version, compared to minian
image
image

**Comment by doriclenses** [comment]
> > > > > Numpy had some update that make caiman preview not working.
> > > > > I updated the functions that are being used for the preview from the updated version of caiman package.
> > > > 
> > > > 
> > > > Do we have to update numpy? Shouldn't we use the version of numpy that is in the requirements of caiman?
> > > 
> > > 
> > > I re installed caiman in anaconda. I did that because I thought it could fixe some problem we had with caiman and also I thought it was a good opportunity to have the last version of caiman ( It was a sort of "mistake" to do that) so now it is using newer version of numpy.
> > > Also before doing the pullrequest I did try to downgrade numpy and I couldn't do that because of dependency and so on.... so I don't really want to start to change numpy version and have thing broken in caiman just because of that.
> > 
> > 
> > Ok, the caiman library changed the requirements for numpy? Or just numpy released a new version?
> 
> I see, they don't have any requirements for version, compared to minian image image

Maybe, we can fix them ourselves to avoid problems in the future

**Pull request merged**